php/files.php

```diff
@@ -87,10 +87,27 @@
         http_error(500, "PHP script can't create folder " . $uploadDir . ". Check permission, group and owner.");
     }
 
-    if (!check($_FILES) || !check($_FILES['file'])) http_error(400, 'No actual file was given');
+    $errorMessages = [
+        UPLOAD_ERR_INI_SIZE   => "File exceeds allowed size.",
+        UPLOAD_ERR_FORM_SIZE  => "File too large.",
+        UPLOAD_ERR_PARTIAL    => "File only partially uploaded.",
+        UPLOAD_ERR_NO_FILE    => "No file uploaded.",
+        UPLOAD_ERR_NO_TMP_DIR => "Missing temp folder.",
+        UPLOAD_ERR_CANT_WRITE => "Cannot write file to disk.",
+        UPLOAD_ERR_EXTENSION  => "File upload blocked by extension.",
+    ];
+
+    $errorCode = isset($_FILES['file']['error']) ? $_FILES['file']['error'] : UPLOAD_ERR_OK;
```

Comment:
```suggestion
    $errorCode = $_FILES['file']['error'] ?? UPLOAD_ERR_OK;
```
https://www.php.net/manual/en/migration70.new-features.php
supports both php 7 and 8 so it's good on that front